components/Button/button.css

```diff
@@ -0,0 +1,57 @@
+@import url(../../docs-css/reset.css);
+@import url(../common-css/drawer.css);
+@import url(../common-css/container.css);
+@import url(../../style.css);
+
+.btn{
+    /* min-width: 6rem; */
+    width: max-content;
+    font-size: 1rem;
+    font-weight: 500;
+    padding: 0.8rem 1rem 0.8rem 1rem;
+    margin: 1rem;
+    border: none;
+    border-radius: 3px;
+    background-color: var(--primary);
+    color: var(--font-secondary);
```

Comment:
Variable naming could be better. The first look at the variable made me think of font family.